src/NzbDrone.Core.Test/ParserTests/QualityParserFixture.cs

```diff
@@ -279,6 +279,7 @@ public void should_parse_webrip1080p_quality(string title, bool proper)
         [TestCase("[HorribleSubs] Series Title! S01 [Web][MKV][h264][2160p][AAC 2.0][Softsubs (HorribleSubs)]", false)]
         [TestCase("Series Title S02 2013 WEB-DL 4k H265 AAC 2Audio-HDSWEB", false)]
         [TestCase("Series.Title.S02E02.This.Year.Will.Be.Different.2160p.WEB.H.265", false)]
+        [TestCase("Series.Title.S02E04.German.Dubbed.DL.AAC.2160p.DV.HDR.WEB.HEVC-GROUP", false)]
```

Comment:
Do you have an example for AVC as well? If not then I think just HEVC makes sense.

Reply:
I currently do not, but I will check if i can find an AVC example aswell!

Reply:
That'd be great, just want to eliminate a potential regression in the future if this regex changes and all tests pass, but it breaks something for an untested case.

Reply:
Found and added a AVC test example!